packages/handlersjs-core/lib/logging/logger.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { Logger } from './logger';
+import { LoggerLevel } from './logger-level';
+
+describe('Logger', () => {
+
+  class MockLogger extends Logger {
```

Comment:
You can simply mock this as an object with those functions.

Reply:
If I used a mocked object with jest functions I still had to implement all the functionality of the functions since it's an abstract class. Resulting in more code than this way? Am I missing something here?

Reply:
A jest mock implementation could just as well contain `throw new Error('Method not implemented.');` if you don't need it.